config/default/field.field.block_content.uiowa_collection.field_uiowa_collection_items.yml

```diff
@@ -25,52 +25,13 @@ settings:
     target_bundles:
       uiowa_collection_item: uiowa_collection_item
     target_bundles_drag_drop:
-      accordion:
-        weight: 17
-        enabled: false
-      accordion_item:
-        weight: 18
-        enabled: false
-      articles:
-        weight: 19
-        enabled: false
-      blockquote:
-        weight: 20
-        enabled: false
-      card:
-        weight: 21
-        enabled: false
-      carousel:
-        weight: 22
-        enabled: false
-      carousel_image:
-        weight: 23
-        enabled: false
-      events:
-        weight: 24
-        enabled: false
-      featured_content:
-        weight: 25
-        enabled: false
-      image_gallery:
-        weight: 26
-        enabled: false
-      people:
-        weight: 27
-        enabled: false
-      section:
-        weight: 28
-        enabled: false
-      text:
-        weight: 29
-        enabled: false
```

Comment:
I'm not sure what is going on here. @cory-skeers Can you provide any insight?

Reply:
these are v2 bundles. so they don't exist

Reply:
I wonder how they ended up in here in the first place.